clusterConfigSchema.yml

```diff
@@ -17,6 +17,7 @@ clusters:
       type: "<vm / physical / bf>"
```

Comment:
We are fixing this in a different PR. Can you push your PR without his commit? We are going to rename bmc_ip to either bmc or something that shows the fact that it can be both the host name or ip. Also, we are making it mandatory since the previous default is actually not that useful.